.circleci/config.yml

```diff
@@ -1708,6 +1710,8 @@ workflows:
       - cannon-stf-verify:
           requires:
             - go-mod-download
+          context:
+            - slack
```

Comment:
Should we remove this from the dependencies for the op-e2e-cannon-tests as well? Then the two jobs can run in parallel.